src/components/notes/list.jsx

```diff
@@ -11,25 +12,43 @@ import {
 } from 'cozy-ui/react/Table'
 import Empty from 'cozy-ui/react/Empty'
 import Stack from 'cozy-ui/react/Stack'
+
 import { translate } from 'cozy-ui/react/I18n'
 import { withBreakpoints } from 'cozy-ui/transpiled/react'
-
+import { withClient } from 'cozy-client'
 import Add from 'components/notes/add'
 import icon from 'assets/icons/icon_note_empty.svg'
 import 'components/notes/list.styl'
+import useReferencedFolderForNote from 'components/notes/hooks/useReferencedFolderForNote'
 
-const EmptyComponent = translate()(({ t }) => (
-  <div className="empty">
-    <Empty
-      id="empty"
-      icon={icon}
-      title={t('Notes.Empty.welcome')}
-      text={t('Notes.Empty.after_created')}
-    >
-      <Add className="u-mt-1" />
-    </Empty>
-  </div>
-))
+const EmptyComponent = translate()(
+  withClient(({ t, client }) => {
+    const { notesFolder } = useReferencedFolderForNote(client)
+    return (
+      <div className="empty">
```

Comment:
nit: Not a priority today, but I see this component (large image then button) in multiple apps, and it may be a good idea to have it in cozy-ui in the future

Reply:
[It exists](https://docs.cozy.io/cozy-ui/react/#!/Empty), it's what we're using here. But because of the markdown here there's some extra complexity.

Reply:
I think @edas talks about the need of the 500px height and translateZ etc. It should be part of the default component since we use that every time